step_exec_lib/utils/processes.py

```diff
@@ -22,7 +23,7 @@ def run_and_handle_error(args: List[str], expected_error_text: str, **kwargs: An
     if "text" not in kwargs:
         kwargs["text"] = True
 
-    run_res = subprocess.run(args, **kwargs, check=False, stdout=subprocess.PIPE, stderr=subprocess.STDOUT)  # nosec
+    run_res = subprocess.run(args, **kwargs, check=False, stdout=sys.stdout, stderr=sys.stderr)  # nosec
```

Comment:
Hmm, this changes the behaviour a lot... previously stderr was piped together with stdout, now they are 2 separate streams... we might depend on the old behaviour somewhere